docs/reference/release-notes/highlights.asciidoc

```diff
@@ -15,6 +15,23 @@ For detailed information about this release, see the <<es-release-notes>> and
 // should be featured in the Stack Installation and Upgrade Guide:
 
 // tag::notable-highlights[] 
+[discrete]
+=== 7.x REST API compatibility
+
+8.0 introduces several breaking changes to the {es} REST APIs. While it's
+important to update your application to account for these changes, finding and
+updating every API call in a single upgrade can be painful and error-prone. To
+make this process easier, we've added support for 7.x compatibility headers to
+our REST APIs. These optional headers let you make 7.x-compatible
```

Comment:
Should we clarify with "In many cases" to help prevent the assumption for all cases ?

Reply:
Good point. I added a caveat with your wording in https://github.com/elastic/elasticsearch/pull/83614/commits/fb95f847ab15f6b207a96edb27e29ce204219774. Thanks!